projects/knora-ui/src/lib/viewer/values/date-value/date-value.component.spec.ts

```diff
@@ -175,7 +175,7 @@ describe('DateValueComponent', () => {
 
       expect(testHostComponent.inputValueComponent.mode).toEqual('read');
 
-      expect(valueReadModeNativeElement.innerText).toEqual('Date: 5/13/2018');
```

Comment:
I was wondering whether we have to mock the date pipe in the spec.

Reply:
to be figured out in https://dasch.myjetbrains.com/youtrack/issue/DSP-98